src/classad/classad/fnCall.h

```diff
@@ -161,19 +161,13 @@ class FunctionCall : public ExprTree
 	static bool timeZoneOffset(const char*,const ArgumentList&,EvalState&,
 							   Value&);
 	static bool dayTime(const char*,const ArgumentList&,EvalState&,Value&);
-	static bool makeTime(const char*,const ArgumentList&,EvalState&,Value&);
```

Comment:
Did you mean to remove this function and not the one commented out below?

Reply:
`makeTime` isn't defined anywhere, so I did intend to remove it.  While I'm here, I'll also remove the commented out function below it.